packages/@react-spectrum/list/stories/ListView.stories.tsx

```diff
@@ -40,6 +41,18 @@ const items = [
   {key: 'n', name: 'Adobe Acrobat', type: 'file'}
 ];
 
+// taken from https://random.dog/
```

Comment:
anyone know a good place to get our product logos from? i could use them with the new stories Reid introduced